eel_hole/__init__.py

```diff
@@ -153,9 +185,16 @@ def create_app():
     login_manager = LoginManager()
     login_manager.init_app(app)
 
-    datapackage, index = __build_search_index(app)
+    sources = [
```

Comment:
Recommend referring to these as `package_sources` to distinguish them from `sources` as used in resource description metadata -- I ended up down a rabbit hole trying to figure out how folks could filter for pudl parquet tables vs xbrl tables and how folks could search for data source shortcodes that don't appear in the data source titles

then I got to search.py line 104 and it all became clear, but I think that journey is probably unnecessary 😉